### PR TITLE
FUSETOOLS-1982 - adding first cut of rest example

### PR DESCRIPTION
**Pull request by bfitzpat**

### CODE REVIEW

File: templates/medium/cxf-jaxrs/medium-fuse-cxf-jaxrs-spring/pom.xml [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.apache.cxf.archetype</groupId>
+    <artifactId>cxf-jaxrs-service</artifactId>
+    <version>3.0.4.redhat-620133</version>
+    <packaging>war</packaging>
+    <name>Simple CXF JAX-RS webapp service using spring configuration</name>
+    <description>Simple CXF JAX-RS webapp service using spring configuration</description>
+    <properties>
+        <jackson.version>1.8.6</jackson.version>
```
Comment (apupier) [inline templates/medium/cxf-jaxrs/medium-fuse-cxf-jaxrs-spring/pom.xml]: current jackson version is 2.8.0. I think that we can try to update it.
Or is it to be aligned with Fuse version?

File: templates/medium/cxf-jaxrs/medium-fuse-cxf-jaxrs-spring/pom.xml [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+            <scope>test</scope>
+        </dependency>
+    </dependencies>
+    <build>
+        <pluginManagement>
+            <plugins>
+                <plugin>
+                    <groupId>org.apache.tomcat.maven</groupId>
+                    <artifactId>tomcat7-maven-plugin</artifactId>
+                    <version>2.0</version>
```
Comment (apupier) [inline templates/medium/cxf-jaxrs/medium-fuse-cxf-jaxrs-spring/pom.xml]: latest version is 2.2 (released in 2013)

File: templates/medium/cxf-jaxrs/medium-fuse-cxf-jaxrs-spring/pom.xml [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+                        <wtpversion>2.0</wtpversion>
+                    </configuration>
+                </plugin>
+            </plugins>
+        </pluginManagement>
+        <plugins>
+            <plugin>
+                <groupId>org.codehaus.mojo</groupId>
+                <artifactId>build-helper-maven-plugin</artifactId>
+                <version>1.5</version>
```
Comment (apupier) [inline templates/medium/cxf-jaxrs/medium-fuse-cxf-jaxrs-spring/pom.xml]: latest version is 2.11
1.5 was released in 2010

File: templates/medium/cxf-jaxrs/medium-fuse-cxf-jaxrs-spring/pom.xml [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+                        <configuration>
+                            <path>/jaxrs-service</path>
+                        </configuration>
+                    </execution>
+                </executions>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-failsafe-plugin</artifactId>
+                <version>2.8.1</version>
```
Comment (apupier) [inline templates/medium/cxf-jaxrs/medium-fuse-cxf-jaxrs-spring/pom.xml]: latest is 2.19.1
2.8.1 is from 2011

File: templates/medium/cxf-jaxrs/medium-fuse-cxf-jaxrs-spring/src/main/webapp/META-INF/context.xml [hunk 5]
```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Context/>
```
Comment (apupier) [inline templates/medium/cxf-jaxrs/medium-fuse-cxf-jaxrs-spring/src/main/webapp/META-INF/context.xml]: just to be sure, is it normal that the context.xml file is almost empty?

**Comment by apupier** [comment]
I don't understand, there is no Camel Route?

**Comment by apupier** [comment]
a last important point seems that we have a conflict with the JSDT facet which blocks import due to join on build on their side and our side.

**Comment by lhein** [comment]
testPR

**Comment by lhein** [comment]
unblocking template progress

**Pull request merged**